src/common/options/global.yaml.in

```diff
@@ -163,9 +163,9 @@ options:
   type: str
   level: basic
   desc: list of hosts or addresses to search for a monitor
-  long_desc: This is a comma, whitespace, or semicolon separated list of IP addresses
-    or hostnames. Hostnames are resolved via DNS and all A or AAAA records are included
-    in the search list.
+  long_desc: This is a list of IP addresses or hostnames that are separated by
+  commas, whitespace, or semicolons or hostnames. Hostnames are resolved via
```

Comment:
this does not sound right:

> separated by ... or hostnames

Reply:
ah, geez. Thanks, Kefu. That's me and my vim registers.